README.md

```diff
@@ -224,14 +224,13 @@ end
 
 ```
 
-Use `elastic_index.load_from_source = true` to configure an index without ActiveRecord.
+Include `ElasticIndex::LoadFromSource` to configure an index without ActiveRecord.
```

Comment:
Mention that finder methods on the model will get delegated to `ElasticRelation`.